docker-compose.yml

```diff
@@ -31,3 +34,7 @@ volumes:
         driver: local
     netbox-nginx-config:
         driver: local
+    netbox-media-files:
```

Comment:
i cant comment on the line above, as it didnt change, but do we really want to hardcode the nginx port to 80, as that barely ever binds correctly (osx default apache or whatever)

Reply:
That's only the 'container internal port'. `docker-compose` will bind that port to a randomish port.